src/c++/perf_analyzer/README.md

```diff
@@ -164,674 +125,40 @@ We can see from the output that the model was able to complete approximately
-                {
-                    "content": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
-                    "shape": [8,2]
-                }
-        },
-        {
-          "INPUT" :
-                {
-                    "content": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
-                }
-        },
-        {
-          "INPUT" :
-                {
-                    "content": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
-                    "shape": [4,4]
-                }
-        }
-        ...
-      ]
-  }
-```
-
-The following is the example to provide contents as base64 string with explicit shapes:
-
-```
-{
-  "data": [{ 
-      "INPUT": {
-                 "content": {"b64": "/9j/4AAQSkZ(...)"},
-                 "shape": [7964]
-               }},
-    (...)]
-}
-```
-
-Note that for STRING type an element is represented by a 4-byte unsigned integer giving
-the length followed by the actual bytes. The byte array to be encoded using base64 must
-include the 4-byte unsigned integers.
-
-### Output Validation
-
-When real input data is provided, it is optional to request perf analyzer to
-validate the inference output for the input data.
-
-Validation output can be specified in "validation_data" field in the same format
-as "data" field for real input. Note that the entries in "validation_data" must
-align with "data" for proper mapping. The following example describes validation
-data for a model with inputs named, INPUT0 and INPUT1, outputs named, OUTPUT0
-and OUTPUT1, all tensors have shape [4, 4] and data type INT32:
-
-```
-  {
-    "data" :
-     [
-        {
-          "INPUT0" : [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
-          "INPUT1" : [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
-        }
-        ...
-      ],
-    "validation_data" :
-     [
-        {
-          "OUTPUT0" : [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "OUTPUT1" : [2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]
-        }
-        ...
-      ]
-  }
-```
-
-Besides the above example, the validation outputs can be specified in the same
-variations described in "real input data" section.
-
-## Shared Memory
-
-By default perf_analyzer sends input tensor data and receives output
-tensor data over the network. You can instead instruct perf_analyzer to
-use system shared memory or CUDA shared memory to communicate tensor
-data. By using these options you can model the performance that you
-can achieve by using shared memory in your application. Use
---shared-memory=system to use system (CPU) shared memory or
---shared-memory=cuda to use CUDA shared memory.
-
-## Communication Protocol
+<br>
 
-By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
-protocol can be specificed with the -i option. If GRPC is selected the
---streaming option can also be specified for GRPC streaming.
+# Documentation
 
-### SSL/TLS Support
+- [Quick Start](README.md#quick-start)
```

Comment:
Does this link to the section immediately above it?
Ideally the quick start on this page is very slim (just to get started with no commentary about the results), and you link to a more detailed quick start in the docs page.
Thoughts on having a 'features' section like MA does?

Reply:
This quick start is already slim, except for Step 6 where the results are explained. It seems awkward to have steps 1-5 here and then link to some other page that also includes a step 6 results explanation.

I think we should leave this as is and remove the quick start intra-doc hyperlink.

Regarding a features section, let's write that after all the other docs have been filled in, so that links can work correctly or we can change how its written based on how the docs look by the end of this epic.

Reply:
I added a features section with empty links that we can fill in as docs get created.